cocos/core/geometry/aabb.ts

```diff
@@ -76,10 +77,18 @@ export class AABB {
       * 克隆一个 AABB，其会创建出一个值跟输入参数`a`一样的 AABB 实例。注意，每次调用 `clone` 都会创建出新实例，尽可能使用 `copy` 方法以减小 GC 压力。
       * @param a @zh 克隆的目标。 @en The target object to be cloned.
       * @returns @zh 克隆出的 AABB 实例。@en The cloned AABB instance.
+      * @deprecated since v3.8.4. Please use the corresponding instance method instead.
       */
     public static clone (a: AABB | Readonly<AABB>): AABB {
-        return new AABB(a.center.x, a.center.y, a.center.z,
-            a.halfExtents.x, a.halfExtents.y, a.halfExtents.z);
+        warn('The static method AABB.clone has been deprecated. Please use the corresponding instance method instead.');
```

Comment:
The `warn` invocation will increase 206 bytes of package size.
Probably, we could make it wrap around it with  `DEBUG` macros.

Reply:
Could we put it in `cocos/core/deprecated.ts`?

Reply:
deprecated.ts can not distinguish which function is deprecated.

Reply:
> The `warn` invocation will increase 206 bytes of package size. Probably, we could make it wrap around it with `DEBUG` macros.

Yep. Reasonable, it TS compiler can generate warning message in compiling is better.